packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -39,56 +49,154 @@ class PoolMaster extends Component {
+  }
+}
+
+@connectStore({
+  hosts: createGetObjectsOfType('host')
+    .filter((_, { pool }) => ({ $pool: pool.id }))
+    .sort(),
   gpuGroups: createGetObjectsOfType('gpuGroup'),
-})(({ gpuGroups, pool }) => (
-  <div>
-    <h3 className='mb-1'>{_('xenSettingsLabel')}</h3>
-    <Container>
-      <Row>
-        <Col size={3}>
-          <strong>{_('uuid')}</strong>
-        </Col>
-        <Col size={9}>
-          <Copiable tagName='div'>{pool.uuid}</Copiable>
-        </Col>
-      </Row>
-      <Row>
-        <Col size={3}>
-          <strong>{_('poolHaStatus')}</strong>
-        </Col>
-        <Col size={9}>
-          {pool.HA_enabled ? _('poolHaEnabled') : _('poolHaDisabled')}
-        </Col>
-      </Row>
-      <Row>
-        <Col size={3}>
-          <strong>{_('setpoolMaster')}</strong>
-        </Col>
-        <Col size={9}>
-          <PoolMaster pool={pool} />
-        </Col>
-      </Row>
-    </Container>
-    <h3 className='mt-1 mb-1'>{_('poolGpuGroups')}</h3>
-    <Container>
-      <Row>
-        <Col size={9}>
-          <ul className='list-group'>
-            {map(gpuGroups, gpuGroup => (
-              <li key={gpuGroup.id} className='list-group-item'>
-                {renderXoItem(gpuGroup)}
-              </li>
-            ))}
-          </ul>
-        </Col>
-      </Row>
-    </Container>
-    <h3 className='mt-1 mb-1'>{_('supplementalPackPoolNew')}</h3>
-    <Upgrade place='poolSupplementalPacks' required={2}>
-      <SelectFiles
-        onChange={file => installSupplementalPackOnAllHosts(pool, file)}
-      />
-    </Upgrade>
-  </div>
-))
+})
+export default class TabAdvanced extends Component {
+  _closeEditRemoteSyslogForm = () => this.setState({ editRemoteSyslog: false })
+  render () {
+    const { hosts, gpuGroups, pool } = this.props
+    const { editRemoteSyslog } = this.state
+    return (
+      <div>
+        <Container>
+          <Row>
+            <Col>
+              <h3>{_('xenSettingsLabel')}</h3>
+              <table className='table'>
+                <tbody>
+                  <tr>
+                    <th>{_('uuid')}</th>
+                    <Copiable tagName='td'>{pool.uuid}</Copiable>
+                  </tr>
+                  <tr>
+                    <th>{_('poolHaStatus')}</th>
+                    <td>
+                      {pool.HA_enabled
+                        ? _('poolHaEnabled')
+                        : _('poolHaDisabled')}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>{_('setpoolMaster')}</th>
+                    <td>
+                      <PoolMaster pool={pool} />
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>{_('syslogRemoteHost')}</th>
+                    <td>
+                      {map(hosts, host => (
+                        <div key={host.id} className='mb-1'>
+                          <SingleLineRow>
+                            <Col size={1}>
```

Comment:
A `table` or a list (• \<my host\>: \<my syslog host\>) would probably be better here as it would use the space as needed.